content/agentless/agentless_installation_document.md

```diff
@@ -0,0 +1,173 @@
++++
+title = "Chef Infra Agentless Mode example"
+
+[menu.agentless]
+title = "Agentless Mode example"
+identifier = "agentless/example"
+parent = "agentless"
+weight = 20
++++
+
+This document provides a simplified, end-to-end guide to configure and run
+Chef Infra Client 19 RC3 in Agentless (Target Mode) using Habitat (`hab`)
+for managing remote systems.
+
+## Prerequisites
+
+- [Chef Infra Client is installed](/install/)
+- Your `HAB_AUTH_TOKEN` is exported
+- a valid Progress Chef license key
+
+## Create a target credentials file
+
+Create a target credentials file (`~/.chef/target_credentials`) that defines connection settings for each node.
+
+For example:
+
+```toml
+['TARGET_NODE_1']
+host = '<IP_ADDRESS_OR_FQDN>'
+user = '<USERNAME>'
+sudo = true
+key_files = '~/.ssh/key-pair.pem'
+
+['TARGET_NODE_2']
+host = '<IP_ADDRESS_OR_FQDN>'
+user = 'root'
+password = '<PASSWORD>'
```

Comment:
Does the credentials file still require `transport_protocol = 'ssh'`?

Reply:
No. I just verified that it works with and without that flag set. I also tested it by setting the protocol to "smb" (great on Windows, bad on Linux) and it failed as expected. My "host" was an Ubuntu instance